setup.py

```diff
@@ -31,6 +31,6 @@ def readme():
   	"License :: OSI Approved :: MIT License",
   	"Operating System :: OS Independent",
   	"Topic :: Internet",
-  	"Programming Language :: Python :: 2.7"
+  	"Programming Language :: Python :: 3.9.5"
```

Comment:
This is a tiny bug I only noticed when I went to deploy. Apparently this classifier is invalid - you need to specify `3.9`: https://pypi.org/classifiers/